actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -422,8 +422,22 @@ def clear(options = {})
       end
 
       def write(headers)
-        if header = make_set_cookie_header(headers[HTTP_HEADER])
-          headers[HTTP_HEADER] = header
+        # Make a fake response for the cookie to write to:
```

Comment:
```suggestion
        # Make a fake response for the cookie to write to.
```

Reply:
I always use colon when referring to the next line rather than just making a statement about execution. Is that a problem?